client/my-sites/plugins/plugins-list/test/index.jsx

```diff
@@ -2,65 +2,93 @@
  * @jest-environment jsdom
  */
 
-import { shallow } from 'enzyme';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import plugins from 'calypso/state/plugins/reducer';
+import productsList from 'calypso/state/products-list/reducer';
+import { reducer as ui } from 'calypso/state/ui/reducer';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
 import { PluginsList } from '..';
 import { sites, sitesObject } from './fixtures';
 
-describe( 'PluginsList', () => {
-	describe( 'rendering bulk actions', () => {
-		let renderedPluginsList;
-		let plugins;
-		let props;
+const render = ( el, options ) =>
+	renderWithProvider( el, { ...options, reducers: { plugins, ui, productsList } } );
 
-		beforeAll( () => {
-			plugins = [
-				{ sites: sitesObject, slug: 'hello', name: 'Hello Dolly' },
-				{ sites: sitesObject, slug: 'jetpack', name: 'Jetpack' },
-			];
+describe( 'PluginsList', () => {
+	const pluginsList = [
+		{ sites: sitesObject, slug: 'hello', name: 'Hello Dolly' },
+		{ sites: sitesObject, slug: 'arbitrary-plugin', name: 'Arbitrary Plugin' },
+		{ sites: sitesObject, slug: 'jetpack', name: 'Jetpack' },
+	];
 
-			props = {
-				plugins,
-				header: 'Plugins',
-				selectedSite: sites[ 0 ],
-				selectedSiteId: sites[ 0 ].ID,
-				isPlaceholder: false,
-				pluginUpdateCount: plugins.length,
-				inProgressStatuses: [],
-				pluginsOnSites: {
-					hello: plugins[ 0 ],
-					jetpack: plugins[ 1 ],
-				},
-				allSites: sites,
-				hasManagePlugins: true,
-				siteIsAtomic: true,
-			};
-		} );
+	const props = {
+		plugins: pluginsList,
+		header: 'Plugins',
+		selectedSite: sites[ 0 ],
+		selectedSiteId: sites[ 0 ].ID,
+		isPlaceholder: false,
+		pluginUpdateCount: pluginsList.length,
+		inProgressStatuses: [],
```

Comment:
Do we really need all these props? I wonder if we can remove some of them from the test.

Reply:
It seems like this is the case, yes.